.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    tags:
+      - "v1.*"
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+name: Release CLI
+jobs:
+  build:
+    name: Build ${{ matrix.os }}-${{ matrix.architecture }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        architecture: [x86_64, aarch64]
+        include:
+          - os: ubuntu-latest
+            suffix: unknown-linux-gnu
+          - os: macos-latest
+            suffix: apple-darwin
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          target: ${{ matrix.architecture }}-${{ matrix.suffix }}
+      - id: build
+        shell: bash
+        run: |
+          export TARGET=${{ matrix.architecture }}-${{ matrix.suffix }}
+          cargo build --release --target $TARGET
+          cd target/${TARGET}/release
+          tar -cJf goose-${TARGET}.tar.bz2 goose goosed
```

Comment:
should this be lowercase `j` for bzip2? uppercase `J` uses xz compression

https://www.gnu.org/software/tar/manual/html_node/Option-Summary.html#Option-Summary

Reply:
Fixed